src/Confluent.Kafka/Loggers.cs

```diff
@@ -21,8 +21,14 @@
 
 namespace Confluent.Kafka
 {
+    /// <summary>
+    ///     OnLog callback event handler implementations.
```

Comment:
righto, i put the warning here, though it's most appropriate on the actual OnLog events. It's noted on them already, but i'll make it an explicit warning.

delegate is called right away, not enqueued.